docker/generate_build_matrix.py

```diff
@@ -180,7 +180,7 @@ def _write_table(table_tag: str, table_contents: str):
 
 
 def _main():
-    python_pytorch_versions = [('3.11', '2.4.1'), ('3.11', '2.3.1'), ('3.11', '2.2.2')]
+    python_pytorch_versions = [('3.11', '2.5.0'), ('3.11', '2.4.1'), ('3.11', '2.3.1')]
```

Comment:
```suggestion
    python_pytorch_versions = [('3.12', '2.5.0'), ('3.11', '2.4.1'), ('3.11', '2.3.1')]
```
3.12?

Reply:
Approve 2.5.0 first then I'll make 3.12 changes (same PR) and see how many things it breaks